### PR TITLE
Added Custom Services Section in readme

### PR DESCRIPTION
**Pull request by MicheleRezk**

### CODE REVIEW

**Comment by jenschude** [comment]
@celeste-horgan could you have a look at the wording please?

File: README.md [hunk 1]
```diff
@@ -294,3 +294,38 @@ Accessing the command built using LINQ
 ```c#
 var command = ((ClientQueryProvider<Category>) query.Provider).Command;
 ```
+
+## Custom Services
```
Comment (celeste-horgan) [inline README.md]: Question: why does Stores need to be handled differently than other endpoints?

File: README.md [hunk 2]
```diff
@@ -294,3 +294,38 @@ Accessing the command built using LINQ
...
+If you want to support services endpoints that still not covered by the SDK like [Stores](https://docs.commercetools.com/http-api-projects-stores#stores ) endpoint for example, all what you need is to create the domain models like described
```
Comment (celeste-horgan) [inline README.md]:
```suggestion
To support services endpoints the SDK does not currently support, like the [Stores](https://docs.commercetools.com/http-api-projects-stores#stores ) endpoint, to create the domain models as follows: 

```

File: README.md [hunk 3]
```diff
@@ -294,3 +294,38 @@ Accessing the command built using LINQ
...
+        public LocalizedString Name { get; set; }
+    }
+    
+    public class StoreDraft : IDraft<Store>
+    {
+        public string Key { get; set; }
+        public LocalizedString Name { get; set; }
+    }
+```
+Please notice that, you have to add the Endpoint attribute above the main domain model, make it inherit from Resource class, and make the draft model implement IDraft interface.
```
Comment (celeste-horgan) [inline README.md]:
```suggestion
Note that you have to add the Endpoint attribute **above** the main domain model. The Endpoint attribute then must inherit from the `Resource` class, and the draft model should implement the `IDraft` interface.
```

File: README.md [hunk 4]
```diff
@@ -294,3 +294,38 @@ Accessing the command built using LINQ
...
+
+After this you can use them in different commands like this:
```
Comment (celeste-horgan) [inline README.md]:
```suggestion
After this you can use the custom services in different commands as follows:
```

File: README.md [hunk 5]
```diff
@@ -294,3 +294,38 @@ Accessing the command built using LINQ
...
+                Key = $"StoreKey"
+            };
+ var store = this.client.ExecuteAsync(new CreateCommand<Store>(storeDraft)).Result;
+```
+
+or like this:
+ ```c#
+  var store = this.client..ExecuteAsync(new GetByIdCommand<Store>(id)).Result;
+ ```
+ you can also create custom Commands, HttpApiCommands, Request Builders and Additional Parameters builders for them, but don't forget to inject them into services container before calling services.UseCommercetools
```
Comment (celeste-horgan) [inline README.md]:
```suggestion
 You can also create custom Commands, HttpApiCommands, Request Builders and Additional Parameters builders for them, but don't forget to inject them into services container before calling `services.UseCommercetools()`.
```

File: README.md (hunk 1 above)
Comment (MicheleRezk) [reply README.md]: it's just for example as it's not currently supported, it will be handled soon like other endpoints

**Pull request merged**